offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -358,14 +358,14 @@ class QuotationDetails {
                     """
         }
 
-        static String discountItemInHTML(int offerPosition, String description, double discountAmount) {
+        static String discountItemInHTML(int offerPosition, String description, OfferItem offerItem) {
             return """<div class="row product-item">
                         <div class="col-1">${offerPosition}</div>
                         <div class="col-4 ">Discount</div>
-                        <div class="col-1 price-value">-</div>
-                        <div class="col-2 text-center">-</div>
-                        <div class="col-2 price-value">-</div>
-                        <div class="col-2 price-value">-${Currency.getFormatterWithoutSymbol().format(discountAmount)}</div>
+                        <div class="col-1 price-value">${offerItem.getQuantity()}</div>
+                        <div class="col-2 text-center">${offerItem.getUnit()}</div>
+                        <div class="col-2 price-value">${offerItem.getDiscountPerUnit()}</div>
```

Comment:
Where does `discountPerUnit` come from?

Reply:
from the offeritem

Reply:
I am not sure I understand your question

Reply:
Ah I see it is filled in another PR

Reply:
Correct, this is already included in this pr, I merged the work of @wow-such-code